python/tvm/runtime/ndarray.py

```diff
@@ -278,6 +278,38 @@ def empty(shape, dtype="float32", ctx=context(1, 0)):
         ctypes.byref(handle)))
     return _make_array(handle, False, False)
 
+def non_empty(shape, dtype="float32", ctx=context(1, 0)):
```

Comment:
There is  no need to expose the function to here, given it is used internally, we can simply add it to measure methods by 

```
def gen_random_array():
    remote.get_function("random.random_fill")
    nd = nd.empty(shape, dtype, ctx)
    random_fill(nd)
```